### PR TITLE
Fix trivy findings on spring boot image

### PR DESCRIPTION
**Pull request by fabiodmota**
## Description
Fix Spring vulnerability's reported by Trivy Scan

https://github.com/eclipse-tractusx/bpdm/issues/706

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on Keep a Changelog (https://keepachangelog.com/en/1.0.0/),
 
+## [4.1.1] - [unreleased]
```
Comment (nicoprow) [inline CHANGELOG.md:7]: It's great that you thought about the changelog. However we are already on version 5.0.0 unreleased.

**Review by nicoprow — CHANGES_REQUESTED** [review]
Looks good to me, just needs some adjustment to the changelog

**Comment by fabiodmota** [comment]
@nicoprow corrected changelog version on CHANGELOG.md

**Comment by fabiodmota** [comment]
> Looks good to me, just needs some adjustment to the changelog

@nicoprow  corrected changelog version on CHANGELOG.md

**Review by nicoprow — APPROVED** [review]
Please squash the commits or rename the second commit.

**Pull request merged**